docs/udp.rst

```diff
@@ -407,29 +568,66 @@ Evaluate user-defined processes
 
 Let's evaluate the user-defined processes we defined.
 
-Because there is no pre-defined
+Because there is no predefined
 wrapper function for our user-defined process, we use the
 generic :func:`openeo.processes.process` function to build a simple
-process graph that calls our ``fahrenheit_to_celsius`` process::
+process graph that calls our ``Hello_openEO`` process:
+
+.. code-block:: python
+
+    pg = openeo.processes.process(
+            "EVI_timeseries",
+            temporal_interval=["2018-06-15", "2018-06-27"],
+            bbox={"west": 3.7, "south": 51.03, "east": 3.75, "north": 51.05},
+            features = {"type": "FeatureCollection", "features": [
+                        {
+                            "type": "Feature", "properties": {},
+                            "geometry": {"type": "Polygon", "coordinates": [[
+                                [5.1417, 51.1785], [5.1414, 51.1772], [5.1444, 51.1768], [5.1443, 51.179], [5.1417, 51.1785]
+                            ]]}
+                        },
+                        {
+                            "type": "Feature", "properties": {},
+                            "geometry": {"type": "Polygon", "coordinates": [[
+                                [5.156, 51.1892], [5.155, 51.1855], [5.163, 51.1855], [5.163, 51.1891], [5.156, 51.1892]
+                            ]]}
+                        }
+                    ]}
+                )
+
+Alternatively, we can also use :func:`~openeo.rest.connection.Connection.datacube_from_process`
+to construct a :class:`~openeo.rest.datacube.DataCube` object
+which we can process further and download:
 
-    >>> pg = openeo.processes.process("fahrenheit_to_celsius", f=70)
-    >>> pg.print_json(indent=None)
-    {"process_graph": {"fahrenheittocelsius1": {"process_id": "fahrenheit_to_celsius", "arguments": {"f": 70}, "result": true}}}
+.. code-block:: python
 
-    >>> res = connection.execute(pg)
-    >>> print(res)
-    21.11111111111111
+    created_process = conn.datacube_from_process(
+        process_id = "EVI_timeseries",
+        temporal_interval = date,
+        box = bbox,
+        feature = aoi
+    )
+See :ref:`datacube_from_process` for more information on :func:`~openeo.rest.connection.Connection.datacube_from_process`.
 
+Additionally, if you wish to share your UDP with a wider audience,
+you can register it in the `openEO Algorithm Plaza <https://marketplace-portal.dataspace.copernicus.eu/catalogue>`_.
```

Comment:
I'm not sure we should put VITO/CDSE specific things in the generic openEO python client docs